CHANGES/1026.doc.rst

```diff
@@ -0,0 +1,3 @@
+Add documentation for :meth:`/ <yarl.URL.__truediv__>`
+
+-- by :user:`commonism`
```

Comment:
This shouldn't be in imperative mood since it's not a commit message, nor is it a TODO list item. Also, there should be a trailing period. And since it's just one sentence, it's probably not worth separating the byline so much:
```suggestion
The pre-existing :meth:`/ <yarl.URL.__truediv__>` magic method
has been documented in the API reference -- by :user:`commonism`.
```